ansible/www-standalone/tools/promote/promote_release.sh

```diff
@@ -24,6 +24,11 @@ dirmatch=$release_dirmatch
 
 node --no-warnings /home/staging/tools/promote/check_assets.js $srcdir/$2 $dstdir/$2
 
+relative_srcdir=${srcdir/$staging_rootdir/"$site/"}
+relative_dstdir=${dstdir/$dist_rootdir/"$site/"}
+
+node --no-warnings /home/staging/tools/promote/check_r2_assets.mjs $staging_bucket/$relative_srcdir/$2 $prod_bucket/$relative_dstdir/$2
```

Comment:
@targos FYI I have deployed this PR onto www and am fairly confident it should work (without doing an actual release). If this does interfere with https://github.com/nodejs/node/pull/56119 then commenting out these additional lines should restore the old behaviour (i.e. check the DO droplet only).